Doc warnings

## Issue Details
There are warnings in our doc :
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/doc/BGL/Concepts/FaceGraph.h:28: warning: explicit link request to 'face_descriptor' could not be resolved
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/doc/BGL/Concepts/HalfedgeGraph.h:47: warning: explicit link request to 'halfedge_descriptor' could not be resolved
 - /home/cgal-testsuite/cgal_doc_build/CGAL-5.2-I-34/include/CGAL/Variational_shape_approximation.h:1832: warning: end of comment block while expecting command </tt>
## Environment

* CGAL version:5.0

--- a/BGL/doc/BGL/Concepts/FaceGraph.h
+++ b/BGL/doc/BGL/Concepts/FaceGraph.h
@@ -25,7 +25,7 @@ A face descriptor must be `DefaultConstructible`, `Assignable`, `EqualityCompara
 \sa \link PkgBGLConcepts Graph Concepts \endlink
 */
 class FaceGraph {
-  /// Returns a special `boost::graph_traits<HalfedgeGraph>::face_descriptor` object which
+  /// Returns a special `boost::graph_traits<HalfedgeGraph>::%face_descriptor` object which
   /// does not refer to any face of graph object which type is `FaceGraph`.
   static boost::graph_traits<HalfedgeGraph>::halfedge_descriptor null_face();
 };

--- a/BGL/doc/BGL/Concepts/HalfedgeGraph.h
+++ b/BGL/doc/BGL/Concepts/HalfedgeGraph.h
@@ -44,7 +44,7 @@ A model of `HalfedgeGraph` must have the interior property `vertex_point` attach
 \sa \link PkgBGLConcepts Graph Concepts \endlink
 */
 class HalfedgeGraph {
-  /// Returns a special `boost::graph_traits<HalfedgeGraph>::halfedge_descriptor` object which
+  /// Returns a special `boost::graph_traits<HalfedgeGraph>::%halfedge_descriptor` object which
   /// does not refer to any halfedge of graph object which type is `HalfedgeGraph`.
   static boost::graph_traits<HalfedgeGraph>::halfedge_descriptor null_halfedge();
 };

--- a/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
+++ b/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
@@ -1778,7 +1778,7 @@ private:
   }
 
   /*!
-   * @brief subdivides a chord recursively in range `[chord_begin, chord_end).`
+   * @brief subdivides a chord recursively in range `[chord_begin, chord_end)`.
    * @param chord_begin begin iterator of the chord
    * @param chord_end end iterator of the chord
    * @param subdivision_ratio the chord recursive split error threshold

--- a/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
+++ b/Surface_mesh_approximation/include/CGAL/Variational_shape_approximation.h
@@ -1579,7 +1579,7 @@ private:
 
   /*!
    * @brief runs the pseudo Constrained Delaunay Triangulation at each proxy region,
-   * and stores the extracted indexed triangles in @a tris.
+   * and stores the extracted indexed triangles in `tris`.
    * @pre all anchors are found, i.e. all boundary cycles have been visited
    * and attached with at least 3 anchors.
    */
@@ -1778,7 +1778,7 @@ private:
   }
 
   /*!
-   * @brief subdivides a chord recursively in range `[@a chord_begin, @a chord_end).`
+   * @brief subdivides a chord recursively in range `[chord_begin, chord_end).`
    * @param chord_begin begin iterator of the chord
    * @param chord_end end iterator of the chord
    * @param subdivision_ratio the chord recursive split error threshold

